lib/classes/Swift/Plugins/DefaultSenderPlugin.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * This file is part of SwiftMailer.
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Sets the sender of a message if none is set
+ *
+ * @author     Adam Zielinski
+ */
+class Swift_Plugins_DefaultSenderPlugin implements Swift_Events_SendListener
+{
+
+    /**
+     * Email of sender to use in all messages that are sent
+     * without any sender information.
+     *
+     * @var string
+     */
```

Comment:
I would remove the block and add the description in the `@param` of the constructor argument (same for `$defaultSenderName` below).